app/test_common.go

```diff
@@ -148,6 +151,22 @@ func (tApp TestApp) CheckBalance(t *testing.T, ctx sdk.Context, owner sdk.AccAdd
 	require.Equal(t, expectedCoins, coins)
 }
 
+// Create a new auth genesis state from some addresses and coins. The state is returned marshalled into a map.
+func (tApp TestApp) NewAuthGenState(ctx sdk.Context, cdc codec.JSONCodec, addresses []sdk.AccAddress, coins []sdk.Coins) GenesisState {
+	// Create GenAccounts
+	accounts := authtypes.GenesisAccounts{}
+	for i := range addresses {
+		accounts = append(accounts, authtypes.NewBaseAccount(addresses[i], nil, 0, 0))
+		// FundAccount moved to cosmos-sdk/x/bank/testutil in future release
+		if err := simapp.FundAccount(tApp.bankKeeper, ctx, addresses[i], coins[i]); err != nil {
```

Comment:
we have a `FundAccount` util in test_commons too that can be used instead i believe

Reply:
Removed the function that uses it now so no longer needed, but thanks for pointing that out -- I'll use the one we have in test_common if I need it later